Cargo.toml

```diff
@@ -32,6 +33,7 @@ debug = true
 exclude = ["tock"]
 members = [
     "apis/gpio",
+    "apis/alarm",
```

Comment:
Please place `"apis/alarm"` in alphabetical order in this list (i.e. it should be the first entry).